### ISSUE TITLE
pebble app showing someone else's tasks

### ISSUE THREAD

**Issue by roobixshoe** [HabitRPG/habitica#4406]
I am user 70ed393a-99c9-48ba-a263-164dfa8343f5 , running iOS 8.1.2 on my iphone 5s, just started using hRPG today. I noticed there is a Pebble app for this game from developer jollyassjerk called "HABITRPG" on the Pebble App Store.
Setting up the Pebble app, it asks me to enter my UUID and KEY, so I go into my iOS hRPG app and copy the UUID and API token. The Pebble app accepts the information and then loads some one else's tasks into my Pebble...

From the top, my tasks are:
_[removed by admin for reason described in comment below]_

I have reinstalled the hRPG app in my Pebble app, I have restarted my phone and my Pebble... I have emailed my UUID and API Token from the website to my phone and then copy/pasted my info from the email to the hRPG app in my Pebble app... still getting this other user's stuff..
I emailed the dev for the app, but haven't heard anything yet.

**Comment by Alys** [comment]
Thanks very much for reporting this! 

I feared it was a security incident with HabitRPG that was letting you see someone else's tasks, but it turns out that those tasks belong to the developer of the Pebble integration app (the developer posted about the app in a guild yesterday so I know who they are), so it's a flaw in the integration app itself. I'll contact the developer myself to add to what you've already told them.

To protect the privacy of the developer's tasks, I have edited your post to delete them. However thank you VERY much for taking the time to enter them all! Seeing them was what let me work out what was going wrong here.

I expect that the developer will be able to fix this problem fairly quickly. Keep an eye on their own github repo at https://github.com/jollyassjerk/pebbleHabitRPG/
